gensim/test/test_doc2vec.py

```diff
@@ -106,11 +107,13 @@ def test_string_doctags(self):
         # force duplicated tags
         corpus = corpus[0:10] + corpus
 
-        model = doc2vec.Doc2Vec(min_count=1)
+        size = 300
+        model = doc2vec.Doc2Vec(size=size, min_count=1)
```

Comment:
As above, though holding test params constant for at least one cycle was useful, if new-default value would also work as suitable test, best to have it be the value going forward.